package.json

```diff
@@ -11,191 +11,71 @@
-    "autoprefixer": "^10.4.0",
-    "babel-plugin-jsx-remove-data-test-id": "^3.0.0",
-    "classnames": "^2.3.1",
-    "commitizen": "^4.2.4",
-    "cz-conventional-changelog": "^3.3.0",
-    "deep-map-keys": "^2.0.1",
+    "@wordpress/components": "^19.1.4",
+    "@wordpress/compose": "^5.0.6",
+    "@wordpress/data": "^6.1.4",
+    "@wordpress/date": "^4.2.1",
+    "@wordpress/edit-post": "^5.0.16",
+    "@wordpress/element": "^4.0.4",
+    "@wordpress/html-entities": "^3.2.1",
+    "@wordpress/i18n": "^4.2.4",
+    "@wordpress/icons": "^6.1.1",
+    "@wordpress/keycodes": "^3.2.2",
+    "@wordpress/plugins": "^4.0.6",
+    "@wordpress/url": "^3.2.2",
     "eslint": "^7.32.0",
-    "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-import": "^2.25.2",
-    "eslint-plugin-jest": "^25.3.0",
-    "eslint-plugin-jsdoc": "^37.2.0",
-    "eslint-plugin-prettier": "^4.0.0",
-    "eslint-plugin-react": "^7.27.1",
-    "identity-obj-proxy": "^3.0.0",
-    "jest": "^27.4.4",
-    "jest-environment-jsdom": "^27.4.4",
     "lint-staged": "^12.1.2",
-    "lodash": "^4.17.21",
-    "npm-run-all": "^4.1.5",
-    "postcss": "^8.4.5",
-    "postcss-cli": "^9.1.0",
-    "postcss-focus-within": "^5.0.1",
-    "prettier": "npm:wp-prettier@2.2.1-beta-1",
-    "react-router-dom": "^5.3.0",
-    "recursive-copy": "^2.0.13",
-    "rtlcss": "^3.5.0",
-    "sass": "^1.45.0",
-    "semantic-release": "^18.0.1",
-    "semantic-release-version-bump": "^1.4.1",
-    "stylelint": "^13.3.1",
-    "stylelint-config-prettier": "^9.0.3",
-    "stylelint-prettier": "^1.2.0",
-    "webpack": "^5.65.0",
-    "webpack-cli": "^4.7.2"
-  },
-  "rtlcssConfig": {
-    "options": {
-      "autoRename": false,
-      "autoRenameStrict": false,
-      "blacklist": {},
-      "clean": true,
-      "greedy": false,
-      "processUrls": false,
-      "stringMap": []
-    },
-    "plugins": [],
-    "map": false
+    "newspack-scripts": "^1.6.3",
+    "prettier": "npm:wp-prettier@^2.2.1-beta-1",
+    "stylelint": "^13.3.1"
   },
   "browserslist": [
     "extends @wordpress/browserslist-config"
   ],
   "lint-staged": {
     "*.scss": "npm run lint:scss:staged"
   },
-  "jest": {
-    "setupFilesAfterEnv": [
-      "<rootDir>/assets/setupTests.js"
-    ],
-    "testMatch": [
-      "<rootDir>/**/*test.js?(x)",
-      "!**/.eslintrc.*"
-    ],
-    "transform": {
-      "^.+\\.js?$": "babel-jest"
-    },
-    "moduleNameMapper": {
-      "\\.(scss|css)$": "identity-obj-proxy"
-    },
-    "verbose": false,
-    "testEnvironment": "jsdom"
-  },
   "scripts": {
-    "cm": "git-cz",
-    "semantic-release": "semantic-release",
-    "build:js": "calypso-build",
-    "build": "run-p \"build:*\"",
-    "clean": "rm -rf dist/",
-    "dev": "calypso-build --watch",
-    "start": "npm run dev",
-    "test": "jest",
-    "lint": "npm-run-all --parallel lint:*",
+    "cm": "newspack-scripts commit",
+    "semantic-release": "newspack-scripts release --files=newspack.php",
```

Comment:
The output I get on testing step 11 is identical on both `master` and this branch, but it does fail. Assuming this isn't a problem?

```bash
✗ npm run semantic-release -- --no-ci --dry-run

> newspack@1.0.0 semantic-release
> semantic-release "--no-ci" "--dry-run"

[11:30:01 AM] [semantic-release] › ℹ  Running semantic-release version 18.0.1
[11:30:01 AM] [semantic-release] › ✔  Loaded plugin "verifyConditions" from "@semantic-release/npm"
[11:30:01 AM] [semantic-release] › ✔  Loaded plugin "verifyConditions" from "@semantic-release/github"
[11:30:01 AM] [semantic-release] › ✔  Loaded plugin "analyzeCommits" from "@semantic-release/commit-analyzer"
[11:30:01 AM] [semantic-release] › ✔  Loaded plugin "generateNotes" from "@semantic-release/release-notes-generator"
[11:30:01 AM] [semantic-release] › ✔  Loaded plugin "prepare" from "@semantic-release/changelog"
[11:30:01 AM] [semantic-release] › ✔  Loaded plugin "prepare" from "@semantic-release/npm"
[11:30:01 AM] [semantic-release] › ✔  Loaded plugin "prepare" from "semantic-release-version-bump"
[11:30:01 AM] [semantic-release] › ✔  Loaded plugin "prepare" from "@semantic-release/git"
[11:30:01 AM] [semantic-release] › ✔  Loaded plugin "publish" from "@semantic-release/npm"
[11:30:01 AM] [semantic-release] › ✔  Loaded plugin "publish" from "@semantic-release/github"
[11:30:01 AM] [semantic-release] › ✔  Loaded plugin "addChannel" from "@semantic-release/npm"
[11:30:01 AM] [semantic-release] › ✔  Loaded plugin "addChannel" from "@semantic-release/github"
[11:30:01 AM] [semantic-release] › ✔  Loaded plugin "success" from "@semantic-release/github"
[11:30:01 AM] [semantic-release] › ✔  Loaded plugin "fail" from "@semantic-release/github"
[11:30:05 AM] [semantic-release] › ✖  An error occurred while running semantic-release: Error: Command failed with exit code 1: git fetch --tags --update-head-ok https://github.com/Automattic/newspack-plugin.git +refs/heads/release:refs/heads/release
From https://github.com/Automattic/newspack-plugin
 ! [rejected]          v1.25.0    -> v1.25.0  (would clobber existing tag)
    at makeError (/Users/dkoo/Local Sites/newspack/app/public/wp-content/plugins/newspack-plugin/node_modules/execa/lib/error.js:60:11)
    at handlePromise (/Users/dkoo/Local Sites/newspack/app/public/wp-content/plugins/newspack-plugin/node_modules/execa/index.js:118:26)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async fetch (/Users/dkoo/Local Sites/newspack/app/public/wp-content/plugins/newspack-plugin/node_modules/semantic-release/lib/git.js:120:5)
    at async /Users/dkoo/Local Sites/newspack/app/public/wp-content/plugins/newspack-plugin/node_modules/semantic-release/lib/branches/index.js:21:5
    at async pEachSeries (/Users/dkoo/Local Sites/newspack/app/public/wp-content/plugins/newspack-plugin/node_modules/p-each-series/index.js:8:23)
    at async module.exports (/Users/dkoo/Local Sites/newspack/app/public/wp-content/plugins/newspack-plugin/node_modules/semantic-release/lib/branches/index.js:20:3)
    at async run (/Users/dkoo/Local Sites/newspack/app/public/wp-content/plugins/newspack-plugin/node_modules/semantic-release/index.js:57:22)
    at async module.exports (/Users/dkoo/Local Sites/newspack/app/public/wp-content/plugins/newspack-plugin/node_modules/semantic-release/index.js:260:22)
    at async module.exports (/Users/dkoo/Local Sites/newspack/app/public/wp-content/plugins/newspack-plugin/node_modules/semantic-release/cli.js:55:5) {
  shortMessage: 'Command failed with exit code 1: git fetch --tags --update-head-ok https://github.com/Automattic/newspack-plugin.git +refs/heads/release:refs/heads/release',
  command: 'git fetch --tags --update-head-ok https://github.com/Automattic/newspack-plugin.git +refs/heads/release:refs/heads/release',
  escapedCommand: 'git fetch --tags --update-head-ok "https://github.com/Automattic/newspack-plugin.git" "+refs/heads/release:refs/heads/release"',
  exitCode: 1,
  signal: undefined,
  signalDescription: undefined,
  stdout: '',
  stderr: 'From https://github.com/Automattic/newspack-plugin\n' +
    ' ! [rejected]          v1.25.0    -> v1.25.0  (would clobber existing tag)',
  failed: true,
  timedOut: false,
  isCanceled: false,
  killed: false
}
Error: Command failed with exit code 1: git fetch --tags --update-head-ok https://github.com/Automattic/newspack-plugin.git +refs/heads/release:refs/heads/release
From https://github.com/Automattic/newspack-plugin
 ! [rejected]          v1.25.0    -> v1.25.0  (would clobber existing tag)
    at makeError (/Users/dkoo/Local Sites/newspack/app/public/wp-content/plugins/newspack-plugin/node_modules/execa/lib/error.js:60:11)
    at handlePromise (/Users/dkoo/Local Sites/newspack/app/public/wp-content/plugins/newspack-plugin/node_modules/execa/index.js:118:26)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async fetch (/Users/dkoo/Local Sites/newspack/app/public/wp-content/plugins/newspack-plugin/node_modules/semantic-release/lib/git.js:120:5)
    at async /Users/dkoo/Local Sites/newspack/app/public/wp-content/plugins/newspack-plugin/node_modules/semantic-release/lib/branches/index.js:21:5
    at async pEachSeries (/Users/dkoo/Local Sites/newspack/app/public/wp-content/plugins/newspack-plugin/node_modules/p-each-series/index.js:8:23)
    at async module.exports (/Users/dkoo/Local Sites/newspack/app/public/wp-content/plugins/newspack-plugin/node_modules/semantic-release/lib/branches/index.js:20:3)
    at async run (/Users/dkoo/Local Sites/newspack/app/public/wp-content/plugins/newspack-plugin/node_modules/semantic-release/index.js:57:22)
    at async module.exports (/Users/dkoo/Local Sites/newspack/app/public/wp-content/plugins/newspack-plugin/node_modules/semantic-release/index.js:260:22)
    at async module.exports (/Users/dkoo/Local Sites/newspack/app/public/wp-content/plugins/newspack-plugin/node_modules/semantic-release/cli.js:55:5) {
  shortMessage: 'Command failed with exit code 1: git fetch --tags --update-head-ok https://github.com/Automattic/newspack-plugin.git +refs/heads/release:refs/heads/release',
  command: 'git fetch --tags --update-head-ok https://github.com/Automattic/newspack-plugin.git +refs/heads/release:refs/heads/release',
  escapedCommand: 'git fetch --tags --update-head-ok "https://github.com/Automattic/newspack-plugin.git" "+refs/heads/release:refs/heads/release"',
  exitCode: 1,
  signal: undefined,
  signalDescription: undefined,
  stdout: '',
  stderr: 'From https://github.com/Automattic/newspack-plugin\n' +
    ' ! [rejected]          v1.25.0    -> v1.25.0  (would clobber existing tag)',
  failed: true,
  timedOut: false,
  isCanceled: false,
  killed: false
}%
```

Reply:
I'd expect a different error (along the lines of "This was triggered on `master` branch but is semantic-release is configured to run on `release`…"). I guess this error might be caused by different state of `release` branch locally on on remote. Anyway, I wouldn't worry about it – the release job is not supposed to succeed neither locally or on `master`.